WordPress/Classes/ViewRelated/Cells/MediaSizeSliderCell.swift

```diff
@@ -0,0 +1,143 @@
+import Foundation
+import UIKit
+
+class MediaSizeSliderCell: UITableViewCell {
+    // MARK: - Default values
+    struct Default {
```

Comment:
This is what i had in mind (sort of!):

```
 struct Model {
    init(value: Int = 1500, minValue : Int = 300, maxValue : Int = 3000, step: Int = 100) {
    }
}
```

And in the container class:

```
private var model = Model()
```

(But that's just a thought!)